helm-charts/k8s-mediaserver/jackett/templates/jackett-resources.yml

```diff
@@ -25,8 +23,8 @@ kind: ConfigMap
 metadata:
   name: jackett-config
 data:
-  PGID: "{{ .Values.general.pgid }}"
-  PUID: "{{ .Values.general.puid }}"
+  PGID: "{{ .Values.global.pgid }}"
```

Comment:
`global` is a helm special keyword here, and it achieves what general used to in the old structure.